src/canvas/DrawButton.jsx

```diff
@@ -2,16 +2,24 @@ import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
 import { Graphics } from '@inlet/react-pixi';
 
-import convertColor from '../utils/convertColor';
+import { convertColor } from '../utils/color';
 import theme from '../context/theme';
+import { getHalf } from '../utils/calcLayout';
 
 export default function DrawButton({ log, index, clicked, onClickHandler }) {
   const buttonGraphics = useCallback(
     (button) => {
       button.clear();
       button.alpha = index === clicked ? 1 : 0;
-      button.beginFill(convertColor(theme.background.aqua));
-      button.drawRect(0, index * 50, 10000, 50);
+      button.beginFill(convertColor(log.color), 0.5);
```

Comment:
```suggestion
      button.beginFill(convertColor(log.color), 0.5);
```

여기 2번째 인자가 알파값인 걸로 아는데 상단 13번째 라인의 button.alpha 와는 다른건가요

Reply:
같은 값인 것 같습니다. 수정하겠습니다.